release-info.json.sample

```diff
@@ -0,0 +1,8 @@
+{
+	"description": "Bringing a minimal OpenShift 4.1 or newer cluster to your local laptop or desktop computer.",
+	"version": {
+		"crcVersion": fillCrcVersion,
+		"gitSha": fillGitCommitSha
+	},
+	"openshiftVersion": fillOpenShiftVersion
```

Comment:
Mostly personal preference, but I'd add obvious markers for whta's going to be substituted, for example @CRC_VERSION@, @GIT_COMMIT_SHA@, @OPENSHIFT_VERSION@
A bit odd that "openshiftVersion" is not part of the "version" struct.

Reply:
Why a description if this is not intended to be read by a human?

Reply:
Can remove the description..